doc/admin/qserv-ingest/ingest.yaml

```diff
@@ -0,0 +1,69 @@
+version: 15
+ingest:
+    http:
+        # Optional
```

Comment:
If you say `Optional' then you may also mention what is the value of the default timeout, if it's some pre-defined long value, or no timeout (waiting indefinitely).

Alternatively, you may want to explain this in some RST file instead of this YAML file.